packages/core/core.ts

```diff
@@ -314,13 +312,23 @@ export function asFindingAwareFuzzFn(
 	originalFuzzFn: fuzzer.FuzzTarget,
 	dumpCrashingInput = true,
 ): FindingAwareFuzzTarget {
+	function printAndDump(error: unknown): void {
+		cleanErrorStack(error);
+		if (
+			!(error instanceof FuzzerSignalFinding) ||
```

Comment:
What would happen when `error` is an instance of `FuzzerSignalHandling`?

Reply:
Only `FuzzerSignalHandling` errors with `exitCode` `Ok` should not be printed and dumped, all others should. Change the check?

Reply:
Yeah I think changing it to `!(error instanceof FuzzerSignalFinding && error.exitCode === FuzzingExitCode.Ok)` would make it easier to understand.

Reply:
Done.